nipype/interfaces/fsl/ICA_AROMA.py

```diff
@@ -0,0 +1,112 @@
+                    'functional data to structural space (.mat file)')
+    fnirt_warp_file = File(exists=True, argstr='-warp %s', xor=['feat_dir'],
+                           desc='File name of the warp-file describing '
+                           'the non-linear registration (e.g. FSL FNIRT) '
+                           'of the structural data to MNI152 space (.nii.gz)')
+    motion_parameters = File(exists=True, mandatory=True,
+                             argstr='-mc %s', xor=['feat_dir'],
+                             desc='motion parameters file')
+    denoise_type = traits.Enum('nonaggr', 'aggr', 'both', 'no', usedefault=True,
+                               mandatory=True, argstr='-den %s',
+                               desc='Type of denoising strategy: '
+                               '-none: only classification, no denoising '
+                               '-nonaggr (default): non-aggresssive denoising, i.e. partial component regression '
+                               '-aggr: aggressive denoising, i.e. full component regression '
+                               '-both: both aggressive and non-aggressive denoising (two outputs)')
+
+class ICA_AROMAOutputSpec(TraitedSpec):
+    aggr_denoised_file=File(exists=True, 
+                           desc='if generated: aggressively denoised volume')
+    nonaggr_denoised_file=File(exists=True,
+                              desc='if generated: non aggressively denoised volume' )
+    out_dir = Directory(exists=True, 
+                        desc='directory contains (in addition to the denoised files): '
+                        'melodic.ica + classified_motion_components + '
+                        'classification_overview + feature_scores + melodic_ic_mni)')
+   
+class ICA_AROMA(CommandLine):
+    """
+    Interface for the ICA_AROMA.py script (v0.3 beta).
+    
+    ICA-AROMA (i.e. 'ICA-based Automatic Removal Of Motion Artifacts') concerns 
+    a data-driven method to identify and remove motion-related independent
+    components from fMRI data. To that end it exploits a small, but robust
+    set of theoretically motivated features, preventing the need for classifier
+    re-training and therefore providing direct and easy applicability. 
+
+
+    See link for further documentation: https://github.com/rhr-pruim/ICA-AROMA
+
+
+
+    Example
+    -------
+
+    >>> from nipype.interfaces.fsl import ICA_AROMA 
+    >>> from nipype.testing import example_data
+    >>> AROMA_obj = ICA_AROMA.ICA_AROMA()
+    >>> AROMA_obj.inputs.in_file = 'functional.nii'
+    >>> AROMA_obj.inputs.mat_file = 'func_to_struct.mat'
+    >>> AROMA_obj.inputs.fnirt_warp_file = 'warpfield.nii'
+    >>> AROMA_obj.inputs.motion_parameters = 'functional.par'
+    >>> AROMA_obj.inputs.mask = 'mask.nii.gz'
+    >>> AROMA_obj.inputs.denoise_type = 'both'
+    >>> AROMA_obj.inputs.out_dir = 'ICA_testout'
+    >>> AROMA_obj.cmdline # doctest: +ALLOW_UNICODE
```

Comment:
How do I get rid of the u'string' output? (and I need to to change the motion parameter file)